### ISSUE TITLE
warn for aria-readonly="true" on any element that also has a readonly attribute

### ISSUE THREAD

**Issue by carmacleod** [IBMa/equal-access#352]
Steps:
- https://w3c.github.io/html-aria/tests/readonly-test.html
- run accessibility checker
- note that for Test 1 (aria-readonly="true" with readonly) input type=password, date, month, week, time are flagged as "The WAI-ARIA role or attribute 'aria-readonly' is not valid for the element `<input>`" violations
- instead, should flag all 13 inputs in the test with a warning saying authors shouldn't use aria-readonly="true" with readonly

**Comment by MHoov** [comment]
As of June 1
- [x] general rule description (SME & Dev)
- [x] testcases written (Dev)
- [ ] rule developed  (Dev)
- [ ] message written (SME)
- [ ] help written (SME)

**Comment by marcjohlic** [comment]
Awaiting results from rule research  https://app.zenhub.com/workspace/o/ibma/equal-access/issues/708

**Comment by shunguoy** [comment]
close as fixed per #800

**Comment by eirckr** [comment]
@shunguoy the recommendation to have this as warnings not as violation. 
<img width="1444" alt="Screen Shot 2022-05-03 at 11 56 28 AM" src="https://user-images.githubusercontent.com/62436670/166502158-e5b81f60-dfa9-498c-8b9b-ff3f8814c955.png">

**Comment by shunguoy** [comment]
@eirckr   this error message is not the one fixed by the PR.  The test case is from w3, each one might have multiple violations or warnings. Also, note, when multiple violations or warnings apply to one element, not all the violations/warnings, are shown. . please look at the correct message for those:  it should be something like "The ARIA attribute \"{0}\" should not overlap is redundant with the HTML attribute".  Or "The ARIA attribute \"{0}\" conflicts with the HTML attribute.  Please check the original issue and comments in the box linked to the issue.

**Comment by eirckr** [comment]
Validated
